.gitmodules

```diff
@@ -1,9 +1,11 @@
 [submodule "panda"]
 	path = panda
-	url = ../../commaai/panda.git
+	url = https://github.com/Smartype/panda.git
+	branch = toyota_nodsu_long
```

Comment:
This isn't the correct way to do it. You should open a PR in the submodule's repository, in your local openpilot submodule's repo add your remote and checkout the PR's commit, then push it.